Neos.Neos/Classes/Command/NodeCommandControllerPlugin.php

```diff
@@ -324,4 +341,34 @@ protected function createContext($workspaceName, array $dimensions)
 
         return $this->contextFactory->create($contextProperties);
     }
+
+    /**
+     * Attaches a new event handler
+     *
+     * @param string $eventIdentifier one of the EVENT_* constants
+     * @param \Closure $callback a closure to be invoked when the corresponding event was triggered
+     * @return void
+     */
+    public function on($eventIdentifier, \Closure $callback)
+    {
+        $this->eventCallbacks[$eventIdentifier][] = $callback;
+    }
+
+    /**
+     * Trigger a custom event
+     *
+     * @param string $eventIdentifier one of the EVENT_* constants
+     * @param array $eventPayload optional arguments to be passed to the handler closure
+     * @return void
+     */
+    protected function dispatch($eventIdentifier, ...$eventPayload)
```

Comment:
```suggestion
    protected function dispatch(string $eventIdentifier, array ...$eventPayload): void
```

Reply:
`...` is the type hint, it doesn't need an additional type hint. The other ones make sense though